main.go

```diff
@@ -82,6 +82,672 @@ var (
 	masterMetrics = []snapshotMetric{
 		snapshotMetric{
 			desc: prometheus.NewDesc(
+				"mesos_master_cpus_percent",
+				"Percent of CPUs used as a fraction of 1.",
```

Comment:
If you're going to exhaustively list every single metric, you may aswell fix them while you're at it.